h2o-py/tests/testdir_javapredict/pyunit_PUBDEV_7185_GAM_mojo_gaussian.py

```diff
@@ -5,13 +5,15 @@
 from random import randint
 import tempfile
 
+
 def gam_gaussian_mojo():
     h2o.remove_all()
     NTESTROWS = 200    # number of test dataset rows
     PROBLEM="gaussian"
-    params = set_params()   # set deeplearning model parameters
-    df = pyunit_utils.random_dataset(PROBLEM, missing_fraction=0.001)   # generate random dataset
+    params = set_params() 
+    df = pyunit_utils.random_dataset(PROBLEM, seed=2, missing_fraction=0.5) 
```

Comment:
Please remove seed to get a more robust test.

Reply:
What do you mean by robust? I think in software robust means predictable, deterministic. This test is anything but when it fails only once in a while, setting the seed is my attempt at making it more robust, now the test still fails but fails more often, there is more work to be done in making this test more robust.